src/impi/core.cljs

```diff
@@ -126,6 +126,33 @@
         [w h] (:pixi.render-texture/size texture)]
     (.create js/PIXI.RenderTexture w h mode)))
 
+(defmulti draw-shape! (fn [_ v] (v :pixi.shape/type)))
+
+(defmethod draw-shape! :pixi.shape/circle 
+  [graphics {[x y] :pixi.shape/position
+             radius     :pixi.circle/radius}]
+  (.drawCircle graphics x y radius))
+
+(defmethod draw-shape! :pixi.shape/ellipse 
+  [graphics {[x y]          :pixi.shape/position
+             [width height] :pixi.shape/size}] 
```

Comment:
I think in this case `:pixi.shape/size` should be `:pixi.ellipse/axes`. The reason is that this value isn't truly the size, e.g. an ellipse with a "size" of [10 5] will produce a shape 20 across and 10 high. The mathematical term for these values are the major and minor axes (with the major being the longer and minor being the shorter), so `:pixi.ellipse/axes` seems more appropriate.